official/vision/image_classification/augment.py

```diff
@@ -27,7 +27,7 @@
 import tensorflow as tf
 from typing import Any, Dict, List, Optional, Text, Tuple
 
-from tensorflow.python.keras.layers.preprocessing import image_preprocessing as image_ops
```

Comment:
Hi @alanchiao, I finished addressing all the review comments but this one and tested the updated commit (currently dealing with the faff cause by 2FA).

How could I revert to TensorFlow HEAD? Do you mean I should use the tf.keras APIs instead of tf.python.keras ones?
I saw so many tf.python.keras used in this repo tho. Do I need to change all the imports related to this PR?

Thanks!

Reply:
Thanks. Going forward, please resolve any comments that you've resolved in the Github PR.

Just meant to revert this one-line change.

"from tensorflow.python.keras.layers import image_preprocessing as image_ops" works at tf-nightly/etc.

"from tensorflow.python.keras.layers.preprocessing import image_preprocessing as image_ops" works at older versions of TF

Reply:
Making as unresolved - will handle when we merge the PR.

Reply:
I can revert this when I do the final squash of all commits, as well as addressing comments from others (if there are any).